src/tink/querystring/macros/GenParser.hx

```diff
@@ -184,6 +184,13 @@ class GenParser {
         case [[v]]: v.getName().sure();
         case v: f.pos.error('more than one @:formField');
       }
+
+      #if tink_json
+      switch f.meta.getValues(':json') {
+        case [[]]: f.expr = macro tink.Json.parse(this.params.get(prefix));
```

Comment:
Curious, why does it check for a two-dimension array here?

Reply:
https://github.com/haxetink/tink_macro/blob/92f60cbe4f991787ddc8ce5c3ceea9e630a1d8a9/src/tink/macro/Metadatas.hx#L17

Reply:
In short, `@:json(12) @:json(true, false) @:json` will result in `[[12], [true, false], []]` so to speak (except that it's expressions, rather than constants).